src/OrchardCore.Modules/OrchardCore.Contents/Views/CommonPart-Date.Edit.cshtml

```diff
@@ -4,7 +4,7 @@
 <div class="form-group" asp-validation-class-for="LocalDateTime">
     <div class="row col-md-6 col-lg-3">
         <label asp-for="LocalDateTime">@T["Created"]</label>
-        <input asp-for="LocalDateTime" class="form-control" type="datetime-local" />
+        <input asp-for="LocalDateTime" asp-format="{0:yyyy-MM-ddTHH:mm}" class="form-control" type="datetime-local" />
```

Comment:
Is there any space should be added between the date and time?

Reply:
The format has an impact on the value which needs a json format, so with the `T`, not with a space between date and time.